mousestyles/dynamics/__init__.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import print_function, absolute_import, division
+
+import pandas as pd
+import numpy as np
+from mousestyles import data
+
+
+def create_time_matrix(combined_gap=4, time_gap=1, days_index=137):
+    r"""
+    Return a time matrix for estimate the MLE parobability.
+    The rows are 137 mousedays. The columns are time series
+    in a day. The data are the mouse activity at that time.
+    0 represents IS, 1 represents eating, 2 represents
+    drinking, 3 represents others activity in AS.
+
+    Parameters
+    ----------
+    combined_gap: nonnegative float or int
+        The threshold for combining small intervals. If next start time
+        minus last stop time is smaller than combined_gap than combined
+        these two intervals.
+    time_gap: positive float or int
+        The time gap for create the columns time series
+    days_index: nonnegative int
+        The number of days to process, from day 0 to day days_index.
+
+    Returns
+    -------
+    time: Pandas.DataFrame
+        a matrix represents the activity for a certain
+        mouse day and a certain time.
+
+    Examples
+    --------
+    >>> time = create_time_matrix(combined_gap=4, time_gap=1).iloc[0, 0:10]
+    >>> strain    0
```

Comment:
Inside of the example, strain 0 mouse 0 day 0, does that mean it is for the first mouse of the first strain and for the first mouse day?